container/container_unix.go

```diff
@@ -345,7 +346,10 @@ func (container *Container) UnmountVolumes(forceSyscall bool, volumeEventLog fun
 
 		if volumeMount.Volume != nil {
 			if err := volumeMount.Volume.Unmount(volumeMount.ID); err != nil {
-				return err
+				logrus.Errorf("%s Unmount failed %v", volumeMount.Volume.Name(), err)
+				// on error keep unmounting the rest of the volumes. Fixed in docker 1.13
+				unmountError = err
+				continue
 			}
 			volumeMount.ID = ""
```

Comment:
What does setting the ID to "" actually do. Does it remove the mount from docker volume ls? Just want to make sure it has no utility